app/src/main/java/com/hedvig/app/feature/offer/OfferViewModel.kt

```diff
@@ -1,11 +1,13 @@
 package com.hedvig.app.feature.offer
 
+import androidx.annotation.StringRes
```

Comment:
Are these automatic checks by ktlint fixed if you run 
ktlint -F "\!**/generated/**" "**/src/**/*.kt*”
on your terminal?

Reply:
I realise now that on the PR view it's not visible, but while reviewing ktlint was complaining about the ordering of some imports, and I wonder if that's a case of IntelliJ auto-formatter formatting them differently from what ktlint likes

Reply:
My IDE does not complain on the ordering here 🤔 Do we have different settings?

Reply:
Right, my IDE wasn't either, but while being on the review mode inside GitHub I saw a bunch of warnings like in this picture here
![image](https://user-images.githubusercontent.com/44558292/133963236-bcd66576-9d9e-47ba-b40f-64cd1a9b49f7.png)
but now they are gone. I think the IDE doesn't complain about the ordering the same way that ktlint does. But I would hope the Ktlint IDE plugin should indicate these as warnings by default too 🤔